### PR TITLE
[DevTools] Add React Devtools Extension Startup and File Structure to OVERVIEW.md

### PR DESCRIPTION
**Pull request by lunaruan**
I made an excalidraw to map out how the React Devtools Extension files are laid out and how initialization works.

Future TODO: Expand this section, add how standalone/inline files are laid out, and make this not an image and/or add accessibility text

### CODE REVIEW

File: packages/react-devtools/OVERVIEW.md [hunk 1]
```diff
@@ -297,3 +297,11 @@ Once profiling is finished, the frontend requests profiling data from the backen
 ### Importing/exporting data
 
 Because all of the data is merged in the frontend after a profiling session is completed, it can be exported and imported (as a single JSON object), enabling profiling sessions to be shared between users.
+
+## Package Specific Details
+
+### Devtools Extension
```
Comment (jstejada) [inline packages/react-devtools/OVERVIEW.md:303]: are these the sections to fill out later?

**Review by jstejada — APPROVED** [review]
yay! thank you!

File: packages/react-devtools/OVERVIEW.md (hunk 1 above)
Comment (lunaruan) [reply packages/react-devtools/OVERVIEW.md:303]: Eep good callout removed the extra title

Comment (bvaughn) [reply packages/react-devtools/OVERVIEW.md:303]: nit: I would maybe combine and reword these titles slightly

I'm not sure this is "package specific" so much as it is...a diagram of how the pieces fit together at a high level. Maybe "Overview diagram" or something?

**Review by bvaughn — APPROVED** [review]
Nice 😄

**Pull request merged**